inst/rmarkdown/templates/pihph/resources/template.tex

```diff
@@ -0,0 +1,285 @@
+
+\textit{PiHPh} relies on post-publication review of the papers that it publishes. If you have any comments on this piece, please add them to its comments site. You are encouraged to consult this site after reading the paper, as there may be comments from other readers there, and replies from the author. This paper's site is here:
+
+\vspace*{1ex}\noindent \url{\thisdoilink}
+
+\end{tcdoublebox}
+}
+
+\widowpenalty=10000
+\clubpenalty=10000
+
+% === START OF USER CUSTOMIZATION
+
+
+\title{$title$$if(subtitle)$:\\$subtitle$$endif$}
+\author{$for(author)$$author.name$\\\affiliation{$author.affiliation$}$sep$ \and $endfor$}
+
+\authorheader{$if(author-header)$$author-header$$else$$for(author)$$author.name$$sep$, $endfor$$endif$}
+\titleheader{$if(title-header)$$title-header$$else$$title$$endif$}
+
+$if(default-fonts)$
+\setmainfont{Cambria}
+\setlicensefont{Calibri}
+\setlogofont{Lekton}
+$endif$
+$if(alt-fonts)$
+\setmainfont{Caladea}
+\setlicensefont{Carlito}
+\setlogofont{Lekton}
+$endif$
+$if(main)$
+\setmainfont{$main$}
+$endif$
+$if(licensefont)$
+\setlicensefont{$licensefont$}
+$endif$
+$if(logofont)$
+\setlogofont{$logofont$}
+$endif$
+
+
+% If you do not have access to Cambria, you can use the metric-equivalent Caladea: https://fontlibrary.org/en/font/caladea
+% If you do not have access to Calibri, you can use the metric-equivalent Carlito: https://fontlibrary.org/en/font/carlito
+% Lekton available at https://www.fontsquirrel.com/fonts/lekton
+
+% If any of these fonts are not available on your system, feel free to
+% change them here, but of course the final layout will then differ
+
+\usepackage{polyglossia}                   % or use babel if desired
+\setmainlanguage[variant=british]{english} % or use babel if desired
+\usepackage[autostyle, english=british]{csquotes} % remove the english=british option for double quotes
+\usepackage{$gloss$} % for examples, feel free to use anything else
+
+$if(listings)$
+\usepackage{listings}
+$endif$
+$if(lhs)$
+\lstnewenvironment{code}{\lstset{language=Haskell,basicstyle=\small\ttfamily}}{}
+$endif$
+$if(highlighting-macros)$
+$highlighting-macros$
+$endif$
+$if(verbatim-in-note)$
+\usepackage{fancyvrb}
+\VerbatimFootnotes % allows verbatim text in footnotes
+$endif$
+
+% Some recommended packages
+
+\usepackage{booktabs} % for nice tables
+
+$if(biblatex)$
+% BibLaTeX setup, recommended
+\usepackage[backend=biber,
+       style=langsci-unified, % available on CTAN
+       mincrossrefs=50,
+       maxcitenames=3,
+       maxbibnames=50,
+       useprefix=true,
+       doi=false]{biblatex}
+$for(bibliography)$
+\addbibresource{$bibliography$}
+$endfor$
+% remember to include the .bib extension
+% remember to run biber rather than bibtex if using this option
+\AtEveryCitekey{\ifciteseen{}{\defcounter{maxnames}{99}}}
+$endif$
+$if(natbib)$
+% BibTeX setup, deprecated
+\usepackage{natbib}
+\bibpunct[:]{(}{)}{;}{a}{}{,}
+\setlength\bibsep{0pt}
+$endif$
+
```

Comment:
We need to add a part here for Pandoc citation processing. 

Example: 
https://github.com/rstudio/rticles/blob/d89f9c108503001bb00a4dd6a937dccf02006289/inst/rmarkdown/templates/acm/resources/template.tex#L65-L93

This is required to support multiple Pandoc version, and to use your template with a bibliography. If you try to add one, you'll see there will be an error. 

It is good practice to add a dummy example for a bibliography insertion. You would have see the error in the CI

Reply:
7526884